src/pages/PageTwo.vue

```diff
@@ -92,8 +92,8 @@ import MoreAboutKiva from '@/pages/MonthlyGood/MoreAboutKiva';
 import KivaAsExpertContentful from '@/pages/MonthlyGood/KivaAsExpertContentful';
 import FrequentlyAskedQuestions from '@/pages/MonthlyGood/FrequentlyAskedQuestions';
 
-const pageQuery = gql`
-	query ($contentType: String!, $contentKey: String) {
+// ! TODO this query is not working
```

Comment:
@emuvente 

This is the query that does not work. It seems that I can't combine `my` and `contentful` in the same query. The result is a `Network error: Cannot read property 'my' of undefined` Which make no sense to me. This error is the same if this query is executed on prefetch/in methods, server side, client side, etc.

Reply:
@eddieferrer Is this error new or was it failing prior to these changes?

Reply:
When I try to load this page locally, I'm getting the following error for that query, as well as no data returned:
```
Variable "$contentType" is never used
```
That error is happening because Apollo processes the @client query using the local resolvers and then sends the rest of the query to the api, meaning this is what the api sees:
```
query ($contentType: String!, $contentKey: String) {
  my {
    autoDeposit {
      isSubscriber
      __typename
    }
    __typename
  }
  general {
    uiExperimentSetting(key: "mg_hero") {
      key
      value
      __typename
    }
    __typename
  }
}
```
Since only the error and no data is returned for that query, any attempts to read `my` will fail because `data` is `null`. So I think we just aren't able to mix local queries and api queries when using variables in the local queries.

Reply:
thanks @emuvente